test-cases.md

```diff
@@ -0,0 +1,143 @@
+# End-to-End Test Documentation
+
+This document provides an overview of the end-to-end tests written for the extension.
+Each test is briefly described for easy reference.
+
+## Test Cases
+
+### 1. Popup Page Has Title
+
+- **Description**: Tests if the popup page has the correct title.
+- **Test Steps**:
+  - Open the popup page.
+  - Check if the page title matches "Solid Identity Selector".
+
+### 2. Popup Page Without Profiles
+
+- **Description**: Tests the popup page behavior when no profiles are present.
+- **Test Steps**:
+  - Open the popup page.
+  - Verify that a message indicates no selected profile.
+  - Check if the list of identities is empty.
+  - Ensure the 'Add' button is visible.
+
+### 3. Add Button Opens Add-New-Profile Page
+
+- **Description**: Tests if clicking the 'Add' button opens the add-new-profile page.
+- **Test Steps**:
+  - Open the popup page.
+  - Click the 'Add' button.
+  - Verify that the new page title includes "Add a new SOLID profile".
```

Comment:
Solid is written as "Solid", not "SOLID".

Reply:
✔️